guide.txt

```diff
@@ -57,7 +59,7 @@ A Guide to the Armorcurriculum.com user API:
 				}
 			}
 
-		The manager by armorcurriculum.com has it's own branch, the to do list it's own, and so on.
+		The manager by curriculum.jikno.com has it's own branch, the to do list it's own, and so on.
```

Comment:
This sounds a bit funny...
You might want to consider one of the following:
```suggestion
		The manager at manager.jikno.com has it's own branch, the to do list it's own, and so on.
```
or
```suggestion
		The manager by Jikno has it's own branch, the to do list it's own, and so on.
```